[2024-11-02 16:42:51,666.666] INFO [128435118424192] - Started server process [5831]
[2024-11-02 16:42:51,666.666] INFO [128435118424192] - Waiting for application startup.
[2024-11-02 16:42:51,667.667] INFO [128435118424192] - Application startup complete.
[2024-11-02 16:42:51,667.667] INFO [128435118424192] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-02 16:42:55,303.303] INFO [128435118424192] - 127.0.0.1:51752 - "GET / HTTP/1.1" 422
[2024-11-02 16:42:55,307.307] INFO [128435068339904] - b''
[2024-11-02 16:42:55,307.307] INFO [128435068339904] - b'{"detail":[{"type":"missing","loc":["query","payload"],"msg":"Field required","input":null}]}'
[2024-11-02 16:43:18,775.775] INFO [128435118424192] - 127.0.0.1:37842 - "GET /hello HTTP/1.1" 200
[2024-11-02 16:43:18,777.777] INFO [128435068339904] - b''
[2024-11-02 16:43:18,777.777] INFO [128435068339904] - b'{"body":"123"}'
[2024-11-02 16:43:52,979.979] INFO [128435118424192] - Shutting down
[2024-11-02 16:43:53,081.081] INFO [128435118424192] - Waiting for application shutdown.
[2024-11-02 16:43:53,082.082] INFO [128435118424192] - Application shutdown complete.
[2024-11-02 16:43:53,082.082] INFO [128435118424192] - Finished server process [5831]
[2024-11-02 16:46:32,942.942] INFO [128102564425856] - Started server process [6199]
[2024-11-02 16:46:32,942.942] INFO [128102564425856] - Waiting for application startup.
[2024-11-02 16:46:32,943.943] INFO [128102564425856] - Application startup complete.
[2024-11-02 16:46:32,943.943] INFO [128102564425856] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-02 16:46:51,230.230] INFO [128102564425856] - 127.0.0.1:45614 - "GET /hello HTTP/1.1" 200
[2024-11-02 16:46:51,232.232] INFO [128102514558656] - b''
[2024-11-02 16:46:51,232.232] INFO [128102514558656] - b'{"body":"123"}'
[2024-11-02 16:47:27,541.541] INFO [128102564425856] - 127.0.0.1:58994 - "GET /hell HTTP/1.1" 404
[2024-11-02 16:47:27,543.543] INFO [128102514558656] - b''
[2024-11-02 16:47:27,543.543] INFO [128102514558656] - b'{"detail":"Not Found"}'
[2024-11-02 16:48:08,211.211] INFO [128102564425856] - Shutting down
[2024-11-02 16:48:08,317.317] INFO [128102564425856] - Waiting for application shutdown.
[2024-11-02 16:48:08,319.319] INFO [128102564425856] - Application shutdown complete.
[2024-11-02 16:48:08,319.319] INFO [128102564425856] - Finished server process [6199]
INFO:root:b''
INFO:root:b'{"body":"123"}'
[2024-11-02 17:00:36,795.795] INFO [136831155863680] - Started server process [6606]
[2024-11-02 17:00:36,796.796] INFO [136831155863680] - Waiting for application startup.
[2024-11-02 17:00:36,796.796] INFO [136831155863680] - Application startup complete.
[2024-11-02 17:00:36,797.797] INFO [136831155863680] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-02 17:00:43,490.490] INFO [136831155863680] - 127.0.0.1:51610 - "GET /hello HTTP/1.1" 200
[2024-11-02 17:00:43,498.498] INFO [136831104452288] - b''
[2024-11-02 17:00:43,499.499] INFO [136831104452288] - b'{"body":"123"}'
[2024-11-02 18:37:22,683.683] INFO [136831155863680] - Shutting down
[2024-11-02 18:37:22,784.784] INFO [136831155863680] - Waiting for application shutdown.
[2024-11-02 18:37:22,784.784] INFO [136831155863680] - Application shutdown complete.
[2024-11-02 18:37:22,785.785] INFO [136831155863680] - Finished server process [6606]
INFO:root:b''
INFO:root:b'{"body":"123"}'
[2024-11-02 18:39:49,069.069] INFO [135342990558912] - b''
[2024-11-02 18:39:49,069.069] INFO [135342990558912] - b'{"body":"123"}'
[2024-11-02 18:44:40,272.272] INFO [132493577347200] - Started server process [8466]
[2024-11-02 18:44:40,273.273] INFO [132493577347200] - Waiting for application startup.
[2024-11-02 18:44:40,274.274] INFO [132493577347200] - Application startup complete.
[2024-11-02 18:44:40,274.274] INFO [132493577347200] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-03 10:08:55,865.865] INFO [132838568538240] - Started server process [15107]
[2024-11-03 10:08:55,872.872] INFO [132838568538240] - Waiting for application startup.
[2024-11-03 10:08:55,874.874] INFO [132838568538240] - Application startup complete.
[2024-11-03 10:08:55,876.876] INFO [132838568538240] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-03 10:09:28,488.488] INFO [132838568538240] - 127.0.0.1:46636 - "GET /hello HTTP/1.1" 200
[2024-11-03 10:09:28,493.493] INFO [132838517114560] - b''
[2024-11-03 10:09:28,496.496] INFO [132838517114560] - b'{"body":"123"}'
[2024-11-03 10:09:47,009.009] INFO [132838568538240] - Shutting down
[2024-11-03 10:09:47,111.111] INFO [132838568538240] - Waiting for application shutdown.
[2024-11-03 10:09:47,112.112] INFO [132838568538240] - Application shutdown complete.
[2024-11-03 10:09:47,113.113] INFO [132838568538240] - Finished server process [15107]
INFO:root:b''
INFO:root:b'{"body":"123"}'
[2024-11-03 10:12:08,986.986] INFO [133415869284480] - Started server process [16182]
[2024-11-03 10:12:08,989.989] INFO [133415869284480] - Waiting for application startup.
[2024-11-03 10:12:08,991.991] INFO [133415869284480] - Application startup complete.
[2024-11-03 10:12:08,996.996] INFO [133415869284480] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-03 10:12:22,179.179] INFO [133415869284480] - 127.0.0.1:50416 - "GET /hello HTTP/1.1" 200
[2024-11-03 10:12:22,185.185] INFO [133415819019968] - b''
[2024-11-03 10:12:22,186.186] INFO [133415819019968] - b'{"body":"123"}'
[2024-11-22 11:20:03,989.989] INFO [124138046668928] - Started server process [3533]
[2024-11-22 11:20:04,000.000] INFO [124138046668928] - Waiting for application startup.
[2024-11-22 11:20:04,001.001] INFO [124138046668928] - Application startup complete.
[2024-11-22 11:20:04,002.002] INFO [124138046668928] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-11-22 11:20:10,607.607] INFO [124138046668928] - Shutting down
[2024-11-22 11:20:10,709.709] INFO [124138046668928] - Waiting for application shutdown.
[2024-11-22 11:20:10,710.710] INFO [124138046668928] - Application shutdown complete.
[2024-11-22 11:20:10,712.712] INFO [124138046668928] - Finished server process [3533]
[2024-12-11 11:10:19,387.387] INFO [125974460506240] - Started server process [5325]
[2024-12-11 11:10:19,397.397] INFO [125974460506240] - Waiting for application startup.
[2024-12-11 11:10:19,398.398] INFO [125974460506240] - Application startup complete.
[2024-12-11 11:10:19,400.400] INFO [125974460506240] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-12-11 11:10:37,760.760] INFO [125974460506240] - 127.0.0.1:50938 - "GET /hello HTTP/1.1" 200
[2024-12-11 11:10:37,807.807] INFO [125974358263488] - b''
[2024-12-11 11:10:37,807.807] INFO [125974358263488] - b'{"body":"123"}'
[2024-12-11 11:10:51,984.984] INFO [125974460506240] - 127.0.0.1:46242 - "GET /heathcheck HTTP/1.1" 404
[2024-12-11 11:10:51,985.985] INFO [125974358263488] - b''
[2024-12-11 11:10:51,985.985] INFO [125974358263488] - b'{"detail":"Not Found"}'
[2024-12-11 11:11:01,041.041] INFO [125974460506240] - 127.0.0.1:33528 - "GET /healthcheck HTTP/1.1" 200
[2024-12-11 11:11:01,043.043] INFO [125974358263488] - b''
[2024-12-11 11:11:01,043.043] INFO [125974358263488] - b'"OK"'
[2024-12-11 11:11:06,833.833] INFO [125974460506240] - 127.0.0.1:32808 - "GET /database HTTP/1.1" 200
[2024-12-11 11:11:06,833.833] INFO [125974358263488] - b''
[2024-12-11 11:11:06,833.833] INFO [125974358263488] - b'"OK"'
[2024-12-11 11:11:48,673.673] INFO [125974460506240] - Shutting down
[2024-12-11 11:11:48,774.774] INFO [125974460506240] - Waiting for application shutdown.
[2024-12-11 11:11:48,775.775] INFO [125974460506240] - Application shutdown complete.
[2024-12-11 11:11:48,775.775] INFO [125974460506240] - Finished server process [5325]
[2024-12-11 11:11:51,625.625] INFO [128284743299200] - Started server process [6322]
[2024-12-11 11:11:51,626.626] INFO [128284743299200] - Waiting for application startup.
[2024-12-11 11:11:51,627.627] INFO [128284743299200] - Application startup complete.
[2024-12-11 11:11:51,666.666] INFO [128284743299200] - Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
[2024-12-11 11:11:54,400.400] INFO [128284743299200] - 127.0.0.1:59134 - "GET /database HTTP/1.1" 500
[2024-12-11 11:11:54,402.402] ERROR [128284743299200] - Exception in ASGI application
  + Exception Group Traceback (most recent call last):
  |   File "/usr/lib/python3/dist-packages/starlette/_utils.py", line 82, in collapse_excgroups
  |     yield
  |   File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 190, in __call__
  |     async with anyio.create_task_group() as task_group:
  |   File "/usr/lib/python3/dist-packages/anyio/_backends/_asyncio.py", line 678, in __aexit__
  |     raise BaseExceptionGroup(
  | ExceptionGroup: unhandled errors in a TaskGroup (1 sub-exception)
  +-+---------------- 1 ----------------
    | Traceback (most recent call last):
    |   File "/usr/lib/python3/dist-packages/uvicorn/protocols/http/h11_impl.py", line 412, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/fastapi/applications.py", line 289, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/applications.py", line 116, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 186, in __call__
    |     raise exc
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 164, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 189, in __call__
    |     with collapse_excgroups():
    |   File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    |     self.gen.throw(value)
    |   File "/usr/lib/python3/dist-packages/starlette/_utils.py", line 88, in collapse_excgroups
    |     raise exc
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 191, in __call__
    |     response = await self.dispatch_func(request, call_next)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/admin/Documents/Python Projects/Hello world/service.py", line 19, in some_middleware
    |     response = await call_next(request)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 165, in call_next
    |     raise app_exc
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 151, in coro
    |     await self.app(scope, receive_or_disconnect, send_no_error)
    |   File "/usr/lib/python3/dist-packages/starlette/middleware/exceptions.py", line 62, in __call__
    |     await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 55, in wrapped_app
    |     raise exc
    |   File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 44, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/usr/lib/python3/dist-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    |     raise e
    |   File "/usr/lib/python3/dist-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    |     await self.app(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/routing.py", line 746, in __call__
    |     await route.handle(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/routing.py", line 288, in handle
    |     await self.app(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/routing.py", line 75, in app
    |     await wrap_app_handling_exceptions(app, request)(scope, receive, send)
    |   File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 55, in wrapped_app
    |     raise exc
    |   File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 44, in wrapped_app
    |     await app(scope, receive, sender)
    |   File "/usr/lib/python3/dist-packages/starlette/routing.py", line 70, in app
    |     response = await func(request)
    |                ^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 273, in app
    |     raw_response = await run_endpoint_function(
    |                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 190, in run_endpoint_function
    |     return await dependant.call(**values)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/home/admin/Documents/Python Projects/Hello world/service.py", line 40, in database
    |     conn = psycopg2.connect(dbname='tester', user='postgres', password='postgres')
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/lib/python3/dist-packages/psycopg2/__init__.py", line 122, in connect
    |     conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    | psycopg2.OperationalError: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  Peer authentication failed for user "postgres"
    | 
    +------------------------------------

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/uvicorn/protocols/http/h11_impl.py", line 412, in run_asgi
    result = await app(  # type: ignore[func-returns-value]
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    return await self.app(scope, receive, send)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/fastapi/applications.py", line 289, in __call__
    await super().__call__(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/applications.py", line 116, in __call__
    await self.middleware_stack(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 186, in __call__
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/middleware/errors.py", line 164, in __call__
    await self.app(scope, receive, _send)
  File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 189, in __call__
    with collapse_excgroups():
  File "/usr/lib/python3.12/contextlib.py", line 158, in __exit__
    self.gen.throw(value)
  File "/usr/lib/python3/dist-packages/starlette/_utils.py", line 88, in collapse_excgroups
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 191, in __call__
    response = await self.dispatch_func(request, call_next)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/admin/Documents/Python Projects/Hello world/service.py", line 19, in some_middleware
    response = await call_next(request)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 165, in call_next
    raise app_exc
  File "/usr/lib/python3/dist-packages/starlette/middleware/base.py", line 151, in coro
    await self.app(scope, receive_or_disconnect, send_no_error)
  File "/usr/lib/python3/dist-packages/starlette/middleware/exceptions.py", line 62, in __call__
    await wrap_app_handling_exceptions(self.app, conn)(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 55, in wrapped_app
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 44, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/lib/python3/dist-packages/fastapi/middleware/asyncexitstack.py", line 20, in __call__
    raise e
  File "/usr/lib/python3/dist-packages/fastapi/middleware/asyncexitstack.py", line 17, in __call__
    await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 746, in __call__
    await route.handle(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 288, in handle
    await self.app(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 75, in app
    await wrap_app_handling_exceptions(app, request)(scope, receive, send)
  File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 55, in wrapped_app
    raise exc
  File "/usr/lib/python3/dist-packages/starlette/_exception_handler.py", line 44, in wrapped_app
    await app(scope, receive, sender)
  File "/usr/lib/python3/dist-packages/starlette/routing.py", line 70, in app
    response = await func(request)
               ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 273, in app
    raw_response = await run_endpoint_function(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/fastapi/routing.py", line 190, in run_endpoint_function
    return await dependant.call(**values)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/admin/Documents/Python Projects/Hello world/service.py", line 40, in database
    conn = psycopg2.connect(dbname='tester', user='postgres', password='postgres')
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3/dist-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.OperationalError: connection to server on socket "/var/run/postgresql/.s.PGSQL.5432" failed: FATAL:  Peer authentication failed for user "postgres"

